src/app/takken/[slug]/query.ts

```diff
@@ -14,7 +14,7 @@ const GROUP_PAGE_QUERY = gql`
   ${ACTIVITY_BLOCK_FRAGMENT}
 
   query getGroupPage($slug: String) {
-    groups(filters: { slug: { eq: $slug }, leaders: { active: { eq: true } } }) {
```

Comment:
Waarom hal je dit weg? Zo komen ook ex-leiding op de paginas

Reply:
Het klopt inderdaad dat de ex-leiding ook op de pagina's getoond worden, maar dit is niet de correcte implementatie. Ik fix het zometeen.

Reply:
Prima, dan is alles in orde